daisy_workflows/image_import/import_image.sh

```diff
@@ -62,6 +90,7 @@ if [[ "${SOURCEFILE_TYPE}" == "ova" ]]; then
   echo "Import: New source file is ${VMDK}"
 fi
 
+# Third, ensure the output disk has sufficient space to accept the disk image.
```

Comment:
I would not enumerate steps (first, second, third) as they could easily get out of date if we ever add more steps in between.

Reply:
I find it helpful for waypointing, but you have a good point about maintenance. So, yeah, I'm good with removing them.